crates/dojo-world/src/contracts/model_test.rs

```diff
@@ -26,7 +26,9 @@ async fn test_model() {
     let target_dir = manifest_dir.join("target").join("dev");
 
     let ws = scarb::ops::read_workspace(config.manifest_path(), &config).unwrap();
-    let dojo_metadata = dojo_metadata_from_workspace(&ws);
+    let dojo_metadata = dojo_metadata_from_workspace(&ws).expect(
```

Comment:
unwrap in tests.

Reply:
i used expect since it gives a more helpful error message incase code panics there.